docs/developer/release/release_vars.py

```diff
@@ -117,26 +115,19 @@ def execute(command_args, halt_if_fail=True, capture_output=False, working_dir=N
     .decode("utf-8")
 )
 
-ANNO_TOOLS = os.path.join(BUILD_DIR, "annotation-tools")
-ANNO_FILE_UTILITIES = os.path.join(ANNO_TOOLS, "annotation-file-utilities")
+ANNO_FILE_UTILITIES = os.path.join(CHECKER_FRAMEWORK, "annotation-file-utilities")
 
 GIT_SCRIPTS = os.path.join(BUILD_DIR, "git-scripts")
 PLUME_SCRIPTS = os.path.join(BUILD_DIR, "plume-scripts")
 CHECKLINK = os.path.join(BUILD_DIR, "checklink")
 PLUME_BIB = os.path.join(BUILD_DIR, "plume-bib")
 
-BUILD_REPOS = (CHECKER_FRAMEWORK, ANNO_TOOLS)
-INTERM_REPOS = (INTERM_CHECKER_REPO, INTERM_ANNO_REPO)
+BUILD_REPOS = CHECKER_FRAMEWORK
```

Comment:
This seems to change the variable from a tuple to a string.  You can eliminate uses of this variable (and a few other tuples) throughout.